waspc/data/Generator/templates/react-app/src/entities/_entity/actions.js

```diff
@@ -10,6 +12,14 @@ export const add = ({= entityLowerName =}) => ({
   data: {= entityLowerName =}.toData()
 })
 
+/**
+ * @param {{= entity.name =}[]} {= entityLowerName =}s
+ */
+export const set = ({= entityLowerName =}s) => ({
+    type: types.SET,
+    {= entityLowerName =}s: {= entityLowerName =}s.map({= entityLowerName =} => {= entityLowerName =}.toData())
```

Comment:
this will be much nicer when extracted into a variable or sth. but ok for now since it is complicated

Reply:
Ok, motivated by this I introduced shorthands `_e, _entity, _entities, _es, ...`, now it is much more readable!